src/components/common/Plotly.js

```diff
@@ -111,149 +124,199 @@ export default function Plotly() {
 
   if (walkCityData.cityWalk1 !== undefined) {
     walkFill[0] = [
-      <>
+      <div className="walkData">
         <h3>Walkability</h3>
         <p>
           {walkCityData.cityWalk1.city} Score:{' '}
           {walkCityData.cityWalk1.walkability}
-        </p>{' '}
-        <p>How fun it is to walk somewhere yenno?</p>
-      </>,
+        </p>
+      </div>,
     ];
   }
   if (walkCityData.cityWalk2 !== undefined) {
     walkFill[1] = [
-      <>
+      <div className="walkData">
         <h3>Walkability</h3>
         <p>
           {walkCityData.cityWalk2.city} Score:{' '}
           {walkCityData.cityWalk2.walkability}
-        </p>{' '}
-        <p>How fun it is to walk somewhere yenno?</p>
-      </>,
+        </p>
+      </div>,
     ];
   }
   if (walkCityData.cityWalk3 !== undefined) {
     walkFill[2] = [
-      <>
+      <div className="walkData">
         <h3>Walkability</h3>
         <p>
           {walkCityData.cityWalk3.city} Score:{' '}
           {walkCityData.cityWalk3.walkability}
-        </p>{' '}
-        <p>How fun it is to walk somewhere yenno?</p>
-      </>,
+        </p>
+      </div>,
     ];
   }
 
   if (weatherCityData.cityWeather1 !== undefined) {
     weatherFill[0] = [
-      <>
+      <div className="weatherData">
         <h3>Weather</h3>
-        <p>
-          Today's Forecast: {weatherCityData.cityWeather1.description} for{' '}
-          {weatherCityData.cityWeather1.city}
-        </p>
-        <p>{weatherCityData.cityWeather1.description}</p>
-        <p>Cloud Percentage: {weatherCityData.cityWeather1.clouds_all}</p>
-        <p>Feels Like: {weatherCityData.cityWeather1.main_feels_like}</p>
-        <p>Humidity: {weatherCityData.cityWeather1.main_humidity}</p>
-        <p>Pressure: {weatherCityData.cityWeather1.main_pressure}</p>
-        <p>Temperature: {weatherCityData.cityWeather1.main_temp}</p>
-        <p>Max: {weatherCityData.cityWeather1.main_temp_max}</p>
-        <p>Min: {weatherCityData.cityWeather1.main_temp_min}</p>
-        <p>Visibility: {weatherCityData.cityWeather1.visibility}</p>
-        <p>Degree of wind direction: {weatherCityData.cityWeather1.wind_deg}</p>
-        <p>Wind Speed: {weatherCityData.cityWeather1.wind_speed}</p>
-      </>,
+        <div>
```

Comment:
Will need to make this section more dynamic in the future. It works for now but can be cleaned up